packages/ui/components/button/Button.tsx

```diff
@@ -46,6 +46,7 @@ const buttonClasses = cva(
         base: "h-9 px-4 py-2.5 ",
         lg: "h-[36px] px-4 py-2.5 ",
         icon: "flex justify-center min-h-[36px] min-w-[36px] ",
+        iconsm: "flex justify-center p-[4px]",
```

Comment:
Can it be icon-sm? That makes it clearer

Reply:
For sure can